Phoenix/PHShebangPreprocessor.m

```diff
@@ -74,10 +74,24 @@ + (NSString *) process:(NSString *)script atPath:(NSString *)path error:(NSError
         return script;
     }
 
-    // Read past shebang-directive
-    [standardOutputFile readDataOfLength:scanner.scanLocation];
+    // Read past shebang-directive (hold onto it incase it's not the shebang text)
```

Comment:
What if the user has a weirdly formatted shebang, or leading white spaces and/or new lines, and the preprocessor reformats this? Then the command will not match against the output.